config/default.json

```diff
@@ -6,19 +6,13 @@
 	}],
 	"assets_base": "https://s3.amazonaws.com/kinmarketplace-assets/version1/",
 	"db": {
-		"database": "database",
+		"type": "sqlite",
+		"host": "localhost",
+		"port": 3306,
 		"username": "username",
 		"password": "password",
-		"options": {
-			"dialect": "sqlite",
-			"storage": "database.sqlite",
-			"operatorsAliases": false,
-			"pool": {
-				"max": 5,
-				"min": 0,
-				"acquire": 30000,
-				"idle": 10000
-			}
-		}
+		"database": "database.sqlite",
+		"synchronize": true,
```

Comment:
I want `synchronize: false` and to have a separate runnable file that knows how to generate tables.

Reply:
This is part of the `default` config, which (at least for now) is the "dev config".
In a different config file you can set it to `false`, but for development it is easier with it as `true`